hpOneView/resources/resource.py

```diff
@@ -186,6 +191,20 @@ def create(self, resource, timeout=-1):
 
         return self._task_monitor.wait_for_task(task, timeout)
 
+    def create_with_uri(self, resource, uri, timeout=-1):
+        if not resource:
```

Comment:
should you also add a check for the uri? you need both resource and uri to make the post